OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/optimobile/PushRegistration.java

```diff
@@ -46,6 +54,7 @@ public void run() {
 
             switch (api) {
                 case FCM:
+                    this.requestPermissionIfNeeded(context);
```

Comment:
only FCM?

Reply:
Did we confirm if Huawei OS have similar changes? I know they have diverged their OS at some point, but perhaps it is still largely based on Android.

Reply:
Added HMS also.  Never confirmed that, will need a device to try it